container/build_tar.py

```diff
@@ -282,8 +300,7 @@ def main(unused_argv):
     for file_owner in FLAGS.owner_names:
       (f, owner) = file_owner.split('=', 1)
       (user, group) = owner.split('.', 1)
-      if f[0] == '/':
-        f = f[1:]
+      f.lstrip('/')
```

Comment:
doesn't this return the stripped string, vs. modify it in-place?

Reply:
You're absolutely right. Reverted that commit.

It's a bit concerning it didn't break any tests, but that seems outside of the scope of this PR. I'll look into that later.